x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -166,7 +167,40 @@ export async function getKibanaAssets(
   return result;
 }
 
-async function installKibanaSavedObjects({
+const isImportConflictError = (e: SavedObjectsImportFailure) => e?.error?.type === 'conflict';
+/**
+ * retry saved object import if only conflict errors are encountered
+ */
+async function retryImportOnConflictError(
+  importCall: () => ReturnType<SavedObjectsImporterContract['import']>,
+  {
+    logger,
+    maxAttempts = 5,
```

Comment:
I think we probably want more than 5. If there are 100 packages it's possible that 99 hit this case on the first attempt and 98 on the second, etc. Maybe the max attempts should be based on the number of packages we expect users to have installed. I feel like 50 would be more sensible?